app/src/main/java/com/onboarding/parkingsystemjava/mvp/view/ParkingView.java

```diff
@@ -2,16 +2,23 @@
 
 import android.app.Activity;
 import android.content.Context;
+import android.util.Log;
 import android.widget.Toast;
 
 import com.onboarding.parkingsystemjava.R;
 import com.onboarding.parkingsystemjava.databinding.ActivityMainBinding;
+import com.onboarding.parkingsystemjava.fragment.SetParkingLotsDialog;
 import com.onboarding.parkingsystemjava.mvp.contract.MainActivityContract;
 import com.onboarding.parkingsystemjava.mvp.view.base.ActivityView;
 
-public class ParkingView extends ActivityView implements MainActivityContract.MainActivityView {
+import java.util.Objects;
+
+public class
+
+ParkingView extends ActivityView implements MainActivityContract.MainActivityView {
```

Comment:
why do you set a line break here?

Reply:
I really don't know, didn't noticed that, fixed